source/server/options_impl.cc

```diff
@@ -71,6 +71,12 @@ OptionsImpl::OptionsImpl(std::vector<std::string> args,
       " connection to the parent cannot be established. Set this to true to instead continue with"
       " a regular startup, while retaining the new epoch value.",
       cmd, false);
+  TCLAP::SwitchArg skip_hot_restart_parent_stats(
+      "", "skip-hot-restart-parent-stats",
+      "When hot restarting, by default the child instance copies stats from the parent"
+      " instance periodically during the draining period. This can potentially be an"
+      " expensive operation; set this to true to just use new stats.",
```

Comment:
s/just use new stats/reset all stats in child process/

Reply:
Done.